app/service/CountriesService.scala

```diff
@@ -25,37 +25,34 @@ class CountriesService  {
   def getAllCountries: List[Country] = countries
 
   def autoCompleteSynonymCountries(implicit messages: Messages): JsObject = {
-      countries.flatMap(country => country.toNewAutoCompleteJson)
-        .foldLeft(JsObject.empty){(countryList, c) => countryList + (c) }
+    countries.flatMap(country => country.toNewAutoCompleteJson)
+      .foldLeft(JsObject.empty){(countryList, c) => countryList + (c) }
   }
 
   private val countries = List(
-    Country("AE0", "title.abu_dhabi", "AE0", Nil),
+    Country("AE-AZ", "title.abu_dhabi", "AE-AZ", Nil),
     Country("AF", "title.afghanistan", "AF", Nil),
-    Country("AE1", "title.ajman", "AE1", Nil),
+    Country("AE-AJ", "title.ajman", "AE-AJ", Nil),
     Country("AX", "title.aland_islands", "AX", List("Aland Islands")),
-    Country("US0", "title.alaska", "US0", Nil),
+    Country("XQZ", "title.akrotiri", "XQZ", Nil),
     Country("AL", "title.albania", "AL", Nil),
     Country("DZ", "title.algeria", "DZ", Nil),
-    Country("UM0", "title.american_polynesia_and_oceania", "UM0", Nil),
     Country("AS", "title.american_samoa", "AS", Nil),
     Country("AD", "title.andorra", "AD", Nil),
     Country("AO", "title.angola", "AO", Nil),
     Country("AI", "title.anguilla", "AI", Nil),
-    Country("AQ0", "title.antarctica", "AQ0", List("Antartica")),
+    Country("AQ", "title.antarctica", "AQ", List("Antartica")),
     Country("AG", "title.antigua_and_barbuda", "AG", Nil),
     Country("AR", "title.argentina", "AR", Nil),
     Country("AM", "title.armenia", "AM", Nil),
     Country("AW", "title.aruba", "AW", Nil),
-    Country("SH", "title.ascension", "SH", Nil),
+    Country("SH-AC", "title.ascension", "SH-AC", Nil),
     Country("AU", "title.australia", "AU", List("Oz")),
-    Country("AQ1", "title.australian_antarctic_territory", "AQ1", Nil),
     Country("AT", "title.austria", "AT", Nil),
     Country("AZ", "title.azerbaijan", "AZ", Nil),
-    Country("PT1", "title.azores", "PT1", Nil),
     Country("BS", "title.bahamas", "BS", Nil),
     Country("BH", "title.bahrain", "BH", Nil),
-    Country("UM1", "title.baker_island", "UM1", Nil),
+    Country("UM-81", "title.baker_island", "UM1-81", Nil),
```

Comment:
is "UM1-81" a typo here or should the alphaTwoCode be different?

Reply:
Great spot 👍 , thanks! It is a typo :)